src/test/python_tests/requirements.in

```diff
@@ -4,6 +4,7 @@
 # 1) pip install pip-tools
 # 2) pip-compile --generate-hashes --upgrade ./src/test/python_tests/requirements.in
 
+debugpy
```

Comment:
`debutpy` should never have to installed in an environment to debug the server. We inject this from python extension.